src/shape_area_definitions.jl

```diff
@@ -1,30 +1,35 @@
 export area
 export internal_function
 
+
+info(shape::Rectangle) = @info "Calculating area of a rectangle..."
+info1(shape::Hex) = @info "Calculating area of a hex..."
```

Comment:
actually, this can also be info(xxx), no need to name it info1 (becasue of mulitiple dispatch)